### ISSUE TITLE
Various Frontier War bugs

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/NorthstarMods#573]
- [ ] Respawning in Titan causes spawning in South base, when own base is North side
- [ ] ???

**Comment by DBmaoha** [comment]
` Respawning in Titan causes spawning in South base, when own base is North side`
this only happens on boomtown( mp_grave ) right? I think respawn messed up it's spawnpoints, i'll do a quick fix

**Comment by DBmaoha** [comment]
`Zones (friendly and enemy) on Forward Base Kodai are terrible and allow shooting the harvester from further away without getting hit by enemy turrets close to the harvester.`
I think it's good to have a draft on balancing maps, we can add props to the map like fd maps

**Comment by GeckoEidechse** [comment]
> > Respawning in Titan causes spawning in South base, when own base is North side

> this only happens on boomtown( mp_grave ) right? I think respawn reversed it's spawnpoints, i'll do a quick fix

Yes, I only observed it on Boomtown so far. Added an edit to the initial issue now ^^

> > Zones (friendly and enemy) on Forward Base Kodai are terrible and allow shooting the harvester from further away without getting hit by enemy turrets close to the harvester.

> I think it's good to have a draft on balancing maps, we can add props to the map like fd maps

I feel like a better longterm solution would be to change the zones themselves to be smaller and closer to the harvester. I think @F1F7Y was working on reversing the format in order to also add FW to other maps. Not sure what the current state of this is ^^

**Comment by F1F7Y** [comment]
> Respawning in Titan causes spawning in South base, when own base is North side (**EDIT:** Only observed on Boomtown)

We could modify the spawns in the .ents

> Zones (friendly and enemy) on Forward Base Kodai  are terrible and allow shooting the harvester from further away without getting hit by enemy turrets close to the harvester.

I can move the zone it self, but cant easily modify the minimap. This may also require .bsp editing which isnt fun. ( Tbh r5r and tfor have this easier as they have total control over which assets they ship which enables them to edit stuff like this easier )

**Comment by GeckoEidechse** [comment]
> This may also require .bsp editing which isnt fun. ( Tbh r5r and tfor have this easier as they have total control over which assets they ship which enables them to edit stuff like this easier )

I mean we can just ship the modified file, no? Or do you mean something else? ^^

**Comment by F1F7Y** [comment]
> I mean we can just ship the modified file, no? Or do you mean something else? ^^

Yeah. It's just that the download size will go up by a considerable amount.

Thinking about it editing the maps would require us to ship  both the .bsp and edited .bsp_lump ( bsps are around ~300MB ). Ent files are exempt from this.

**Comment by GeckoEidechse** [comment]
Ok yeah, I see now what the issue is.
